lab2/lab2.md

```diff
@@ -0,0 +1,94 @@
+
+These sensors have little chips in them and while they're not that delicate, they do need to be handled properly. Be careful of static electricity when handling them and make sure the power supply is connected up correctly and is between 2.7 and 5.5V DC - so don't try to use a 9V battery!
+
+Remember that you can use anywhere between 2.7V and 5.5V as the power supply. For this example, I'm showing it with a 5V supply but note that you can use this with a 3.3v supply just as easily. No matter what supply you use, the analog output voltage reading will range from about 0V (ground) to about 1.75V. We can connect our sensor to the arduino board without any intermittent connections, so our basic hardware setup will be as follows: 
+
+![TMP36 hardware setup v1](https://github.com/queens-satellite-team/Space-School/blob/b5f909abff8ab9da178e1989abed70ed6fba3ad8/lab2/lab2-images/tmp36-hw-setup-v1.0.png)
+
+And that is it for our hardware requirements! Once you get comfortable with working with the sensor see if you can include an LED light that turns on at certain temperatures with a similar hardware and software setup as we made in [LAB 1](https://github.com/queens-satellite-team/Space-School/tree/main/lab1). 
+
+## Step 2 - Analog Inputs
+Before going forward with the software, we should take a moment to understand how the arduino board will interpret the voltage that it receives from the TMP36 sensor. The arduino uses an analog-to-digital converter (ADC) to map a continuous-time-varying variable such as voltage into what a machine can understand - a digital signal consisting of 0's and 1's. The arduino we are using has a 10-bit ADC. This means that depending on the voltage at the arduino's analog input, the ADC can produce a range of values from 0 all the way to 2^10 or 1024. So this means we have another conversion to do, for a total of two. We must convert the ADC 10-bit value to a voltage value, then this voltage value to a temperature value. We will keep this in mind going forward. 
+
+# Step 3 - Software Implementation 
+So let us begin writing some code! Open the text editor within Tinkercad and copy the lines of code shown below into your own monitor. Our first iteration will be to show what happens 
+
+![TMP36 Software Setup v1](https://github.com/queens-satellite-team/Space-School/blob/30fa3adc33bb848d88b189a41a39b36e814525b6/lab2/lab2-images/tmp36-sw-setup-v1.0.png)
+
+`Line 4: int sensor_pin = 0;`
+- We want to assign a more readable name to the analog input pin on our arduino that we are using. This makes our code easier to read for us and for others, especially when we must revisit our code many months later. 
+
+- This line is seperated as *data type* *variable name* = *value*. The equal sign does not really mean *equal* but more so *assign*. We assign the value of *0* to the variable *sensor_pin* of type *int*. Every variable will have a type, name, and a value. 
+  
```

Comment:
we never introduce what a data type is

Reply:
I will point them to the extra info session